presto-spi/src/main/java/com/facebook/presto/spi/statistics/ConfidenceLevelComparator.java

```diff
@@ -0,0 +1,28 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.facebook.presto.spi.statistics;
+
+import com.facebook.presto.spi.statistics.SourceInfo.ConfidenceLevel;
+
+import java.util.Comparator;
+
+public class ConfidenceLevelComparator
```

Comment:
A separate class for comparison of confidence level seems too much here.

If needed, you can refer to https://github.com/prestodb/presto/blob/master/presto-common/src/main/java/com/facebook/presto/common/plan/PlanCanonicalizationStrategy.java on how to order enum.